internal/go_repository.bzl

```diff
@@ -208,7 +208,24 @@ def _go_repository_impl(ctx):
         "GIT_SSL_CAINFO",
         "GIT_SSH",
         "GIT_SSH_COMMAND",
+        "GIT_CONFIG_COUNT"
     ]
+
+    # Git allows passing configuration through environmental variables, this will be picked
+    # by go get properly: https://www.git-scm.com/docs/git-config/#Documentation/git-config.txt-GITCONFIGCOUNT
+    if 'GIT_CONFIG_COUNT' in ctx.os.environ:
```

Comment:
Please use double quotes around strings for consistency with the rest of the file.